Bottom: ac73ff0db6ce5b9611b2e32509949f660411c62c
Top:    148776fca44aab177be21ac192498e1f182b556d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 14:00:03 +0100

Make Move*ToLayerLowLevel() local to move.c

No point in exporting these functions.


---

diff --git a/src/move.c b/src/move.c
index c8d1938..0c24c90 100644
--- a/src/move.c
+++ b/src/move.c
@@ -435,7 +435,7 @@ MovePolygonPoint (LayerTypePtr Layer, PolygonTypePtr Polygon,
 /* ---------------------------------------------------------------------------
  * moves a line between layers; lowlevel routines
  */
-void *
+static void *
 MoveLineToLayerLowLevel (LayerTypePtr Source, LineTypePtr Line,
 			 LayerTypePtr Destination)
 {
@@ -457,7 +457,7 @@ MoveLineToLayerLowLevel (LayerTypePtr Source, LineTypePtr Line,
 /* ---------------------------------------------------------------------------
  * moves an arc between layers; lowlevel routines
  */
-void *
+static void *
 MoveArcToLayerLowLevel (LayerTypePtr Source, ArcTypePtr Arc,
 			LayerTypePtr Destination)
 {
@@ -640,7 +640,7 @@ MoveLineToLayer (LayerTypePtr Layer, LineTypePtr Line)
 /* ---------------------------------------------------------------------------
  * moves a text object between layers; lowlevel routines
  */
-void *
+static void *
 MoveTextToLayerLowLevel (LayerTypePtr Source, TextTypePtr Text,
 			 LayerTypePtr Destination)
 {
@@ -699,7 +699,7 @@ MoveTextToLayer (LayerTypePtr Layer, TextTypePtr Text)
 /* ---------------------------------------------------------------------------
  * moves a polygon between layers; lowlevel routines
  */
-void *
+static void *
 MovePolygonToLayerLowLevel (LayerTypePtr Source, PolygonTypePtr Polygon,
 			    LayerTypePtr Destination)
 {
diff --git a/src/move.h b/src/move.h
index c4a844b..0547686 100644
--- a/src/move.h
+++ b/src/move.h
@@ -100,9 +100,6 @@ void *MoveObject (int, void *, void *, void *, LocationType, LocationType);
 void *MoveObjectToLayer (int, void *, void *, void *, LayerTypePtr, bool);
 void *MoveObjectAndRubberband (int, void *, void *, void *,
 			       LocationType, LocationType);
-void *MoveLineToLayerLowLevel (LayerTypePtr, LineTypePtr, LayerTypePtr);
-void *MoveTextToLayerLowLevel (LayerTypePtr, TextTypePtr, LayerTypePtr);
-void *MovePolygonToLayerLowLevel (LayerTypePtr, PolygonTypePtr, LayerTypePtr);
 bool MoveSelectedObjectsToLayer (LayerTypePtr);
 
 /* index is 0..MAX_LAYER-1.  If old_index is -1, a new layer is
